flutter/lib/src/sentry_privacy_options.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+import 'package:meta/meta.dart';
+
+import '../sentry_flutter.dart';
+import 'screenshot/masking_config.dart';
+import 'screenshot/widget_filter.dart';
+
+/// Configuration of the experimental privacy feature.
+class SentryPrivacyOptions {
```

Comment:
Is this just moved from SentryReplayOptions or have there been any changes? Just so that I know how detailed the review should be.

Reply:
yes, you are right, I just moved them out of the `SentryReplayOptions`.